### ISSUE TITLE
Accessibility Report: Card Component

### ISSUE THREAD

**Issue by sherakama** [SU-SWS/decanter#316]
Use the AMP tool to scan the [Card](http://decanter.stanford.edu/section-components.html#kssref-components-card) component for accessibility issues and document them to this ticket.

**Comment by sherakama** [comment]
AMP Report: https://stanford.levelaccess.net/public/reporting/view_report.php?report_id=283428

No violations found.

**Comment by sherakama** [comment]
Chrome audit: 100
![Screen Shot 2019-03-13 at 9 11 53 PM](https://user-images.githubusercontent.com/550602/54330868-a7140080-45d4-11e9-8004-c7bcf2504d4c.png)
